pkg/connectors/utils/configuration.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"os"
+)
+
+// Attributes that are defined in a config map or the runtime environment
+const (
+	CatalogConnectorUseTLS  string = "CATALOG_CONNECTOR_USE_TLS"
+	CatalogConnectorUseMTLS string = "CATALOG_CONNECTOR_USE_MTLS"
+	PolicyManagerUseTLS     string = "POLICY_MANAGER_CONNECTOR_USE_TLS"
+	PolicyManagerUseMTLS    string = "POLICY_MANAGER_CONNECTOR_USE_MTLS"
+	CertSecretName          string = "CERT_SECRET_NAME"
+	CertSecretNamespace     string = "CERT_SECRET_NAMESPACE"
+	CACERTSecretName        string = "CACERT_SECRET_NAME"      //nolint:gosec
+	CACERTSecretNamespace   string = "CACERT_SECRET_NAMESPACE" //nolint:gosec
+)
+
+// GetCatalogConnectorUseTLS returns true if data connector communication should use tls.
+func GetCatalogConnectorUseTLS() bool {
+	return os.Getenv(CatalogConnectorUseTLS) == "true"
+}
+
+// GetCatalogConnectorUseMTLS returns true if data connector communication should use mtls.
+func GetCatalogConnectorUseMTLS() bool {
```

Comment:
Why do you need separate env variables ? Each server will get a didicated set of env variables. There is no case that teh same server will get both Catalog and Policy manager env variables.

Reply:
The manager needs to know how to communicate with each server.